common/src/main/java/com/microsoft/identity/common/internal/providers/IdentityProviderFactory.java

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.identity.common.internal.providers;
+
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+
+public abstract class IdentityProviderFactory {
+    public abstract OAuth2Strategy CreateOAuth2Strategy();
```

Comment:
Nitpick: let's lowercase the method name here

Reply:
👍 My roots are showing....